amf-client/shared/src/test/scala/amf/resolution/ProductionResolutionTest.scala

```diff
@@ -56,7 +56,7 @@ class ProductionResolutionTest extends RamlResolutionTest {
       Raml10YamlHint,
       renderOptions = Some(config.renderOptions),
       target = Amf,
-      transformWith = Some(Raml10)
+      transformWith = Some(Amf)
```

Comment:
ProductionResolutionTest: transformWith was adjusted to value that was used previously.